src/docs/metrics/nft-network-profit-loss/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: NFT Network Profit Loss
+author: Filip
+date: 2022-06-29
+description: NFT network realized profit/loss for all NFTs and for each NFT collection separately
+---
+## Definition
+
+
+NFT Network Profit/Loss computes the average profit or loss of all NFTs 
+that change addresses. 
+NFT Collection Profit/Loss computes the average profit or loss of each 
+NFT collection that change addresses daily.
```

Comment:
This metric represents the average profit or loss of each 
NFT collection that is being transacted on a given day.